SettingsList.py

```diff
@@ -4041,6 +4054,83 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             "hide_when_disabled": True,
         },
     ),
+    Combobox(
+        name           = 'empty_dungeons_mode',
+        gui_text       = 'Empty dungeons',
+        default        = 'none',
+        choices        = {
+            'none':       'Off',
+            'specific':   'Specific Dungeons',
+            'count':      'Count',
+        },
+        gui_tooltip    = '''\
+            Empty dungeons are dungeons guaranted to be barren, and whose rewards
+            are given for free to the player before the beginning of the game.
+
+            'None': No dungeon will be empty. All dungeons may contain useful items
```

Comment:
Does the option "None" guarantee that the dungeons will have at least 1 item? Or could the randomizer still randomly roll barren dungeons?

Reply:
No it's a clumsy phrasing from me. 'None' just mean the current way generator works, where some dungeons can be rolled barren. Maybe should I just keep the `No dungeon will be empty.` part and get rid of the rest of the sentence?

Reply:
"No dungeon will be guaranteed empty and no additional Dungeon Rewards will be given for free, even if dungeons are randomly rolled with no items in them." Something like that maybe?